packages/loader/container-loader/src/test/connectionStateHandler.spec.ts

```diff
@@ -382,209 +382,209 @@ describe("ConnectionStateHandler Tests", () => {
 
     it("Should wait for savedEvent before moving to connected state(Client 3) when client 2 " +
         "got disconnected from connecting state", async () => {
-        client.mode = "write";
```

Comment:
nit: Just collapse the string into one line on the previous line then all this indentation can be reverted.  I think I previously disabled the max-len lint rule here (I often do for test files)